File: apps/android/app/src/main/java/com/sofaexchange/domain/model/SofaType.kt
package com.sofaexchange.domain.model

enum class SofaType(val displayName: String) {
    SOFA_BED("Sofa Bed"),
    SIMPLE_SOFA("Simple Sofa");

    companion object {
        fun fromApiValue(value: String): SofaType? = values().find { it.name == value }
    }
}

File: apps/ios/SofaExchange/Domain/Entities/SofaType.swift
import Foundation

enum SofaType: String, Codable, CaseIterable {
    case sofaBed    = "SOFA_BED"
    case simpleSofa = "SIMPLE_SOFA"

    var displayName: String {
        NSLocalizedString("sofa_type_\(rawValue)", comment: "")
    }
}
